src/main/java/org/folio/consortia/service/impl/SharingInstanceServiceImpl.java

```diff
@@ -48,6 +51,8 @@ public class SharingInstanceServiceImpl implements SharingInstanceService {
   private final InventoryService inventoryService;
   private final FolioModuleMetadata folioModuleMetadata;
   private final FolioExecutionContext folioExecutionContext;
+  private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
there's object mapper in application context which could be injected

Reply:
Thank you, corrected.